src/arch/intsimdmatrixrvv.cpp

```diff
@@ -0,0 +1,104 @@
+///////////////////////////////////////////////////////////////////////
+// File:        intsimdmatrixrvv.cpp
+// Description: matrix-vector product for 8-bit data on rvv.
+// Author:      sunyuechi
```

Comment:
Is this your full name? Do you want to add the Chinese name, too (maybe also as the commit author)?

Reply:
sunyuechi is my full name in Chinese(Sun Yuechi)

Reply:
@stweil  It seems I didn’t forget to configure the email, but the commit author shows as hleft <89069008+hleft@users.noreply.github.com> instead of the correct sunyuechi <sunyuechi@iscas.ac.cn>.
After merging, can GitHub easily modify it? If not, it’s fine.

Reply:
I fixed it now.